src/components/Contacts/ContactDetails/ContactDetailsTab/People/Items/EditPersonModal/EditPersonModal.tsx

```diff
@@ -44,80 +50,254 @@ const ShowExtraText = styled(Typography)(() => ({
   fontWeight: 'bold',
 }));
 
+const LoadingIndicator = styled(CircularProgress)(({ theme }) => ({
+  margin: theme.spacing(0, 1, 0, 0),
+}));
+
+const personSchema: yup.SchemaOf<
+  Omit<PersonUpdateInput, 'familyRelationships' | 'id'>
+> = yup.object({
+  firstName: yup.string().required(),
```

Comment:
Is there a way to say the length too? Like yup.string().maxLength() or something. I just briefly looked at the docs but was curious. In Siebel we have lengths restrictions and the message siebel pops up with isn't user friendly, so I want to do as much validation as I can when making the forms in the future.

Reply:
Yep, you can do something like 
```typescript
       firstName: Yup.string()
         .max(15, 'Must be 15 characters or less')
         .required('Required'),
       lastName: Yup.string()
         .max(20, 'Must be 20 characters or less')
```

Reply:
Awesome, thanks!